src/webresource-handler/index.ts

```diff
@@ -63,30 +63,27 @@ export const setupWebresourceHandler = (handler: WebResourceHandler): void => {
 export const getWebresourceHandler = (): WebResourceHandler | undefined => {
 	return configuredWebResourceHandler;
 };
+const notValidUpload = () => Promise.resolve(false);
 
-const isFileInValidPath = async (
-	fieldname: string,
+const getRequestUploadValidator = async (
 	req: Express.Request,
-): Promise<boolean> => {
+	odataRequest: uriParser.ParsedODataRequest,
+): Promise<(fieldName: string) => Promise<boolean>> => {
```

Comment:
Can you make this
```suggestion
): Promise<(fieldName: string) => Resolvable<boolean>> => {
```
and use
```ts
const notValidUpload = () => false;
```
instead please

Reply:
done